libtelemetry/src/main/java/com/mapbox/android/telemetry/CertificateBlacklist.java

```diff
@@ -61,12 +63,25 @@
   };
   private final Context context;
   private final String accessToken;
+  private String userAgent;
   private Logger logger;
+  private OkHttpClient client;
 
-  CertificateBlacklist(Context context, String accessToken) {
+  CertificateBlacklist(Context context, String accessToken, String userAgent) {
     this.context = context;
     this.accessToken = accessToken;
+    this.userAgent = userAgent;
     this.logger = new Logger();
+    this.client = new OkHttpClient();
+  }
+
+  @VisibleForTesting
```

Comment:
should we simply have one constructor and remove the one on line 70? then `@VisibleForTesting` is redundant.